tex/latex/biblatex/lbx/romanian.lbx

```diff
@@ -401,8 +401,8 @@
   pages            = {{paginile}{pp\adddot}},
   column           = {{coloana}{col\adddot}},
   columns          = {{coloanele}{col\adddot}},
-  line             = {{linia}{l\adddot}},
-  lines            = {{liniile}{ll\adddot}},
+  line             = {{linie}{l\adddot}},
+  lines            = {{linii}{ll\adddot}},
```

Comment:
If I understand correctly those two words use a different case(?) than *coloana*, *coloanele*, *versul*, *versuri* etc. now. On first glance that looks inconsistent.

The strings here (`page`/`pages`, `line`/`lines`, ...) are for pinpoint references in the text: "Sigfridsson 1998, `pages` 30-34" and in the bibliography: "In: Journal of Computational Chemistry 19.4 (1998), `lines` 377–395."

The `pagetotal` strings below are used to show the total number of pages in a book.
"Westport, Conn. and London: Greenwood, 2000. 200 `pagestotal`"

Reply:
So it should be: "Sigfridsson 1998, `paginile` 30-34" and in the bibliography: "In: Journal of Computational Chemistry 19.4 (1998), `liniile` 377–395."

Reply:
> The `pagetotal` strings below are used to show the total number of pages in a book.
> "Westport, Conn. and London: Greenwood, 2000. 200 `pagestotal`"

here should be: "Westport, Conn. and London: Greenwood, 2000. 200 `pagini`

Reply:
Does that mean that the state of the translations before this change is correct? In that case could you please roll back the changes (and check the other related translations again just to be sure)?

Reply:
> Does that mean that the state of the translations before this change is correct? In that case could you please roll back the changes (and check the other related translations again just to be sure)?

I reverted in my repo. I'll keep some changes but for compiler I can only match with `redactat de`
We use gender sensitive here: if we consider that the bibliography is showing books, then book is feminin (redactat`ă` / alcătuit`ă`)<- feminin article), but if we consider that is a volume we use neutre (_redactat/aclătuit de_)